alpaca/rest.go

```diff
@@ -429,6 +430,7 @@ func (c *client) GetCalendar(start, end *string) ([]CalendarDay, error) {
 
 // ListOrders returns the list of orders for an account,
 // filtered by the input parameters.
+// Deprecated: This function is deprecated in favor of ListOrdersV2 which contains all possible parameters.
```

Comment:
[minor] Please add a newline before the deprecation notice!

Reply:
Done